### PR TITLE
Np 2524 field transformation

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinTitle.java [hunk 1]
```diff
@@ -15,7 +15,8 @@
     public static final String MAIN_TITLE = "titteltekst";
     public static final String LANGUAGE_CODE = "sprakkode";
     public static final String ABSTRACT = "abstract";
-    public static final String VALUE_OF_ORIGINAL_TITLE_STATUS = "J"; //probably means "Ja"
+    public static final String ORIGINAL_TITLE = "J"; //probably means "Ja"
+    public static final String NOT_ORIGINAL_TITLE = "N"; //probably means "Ja"
```
Comment (brinxmat) [inline cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinTitle.java:19]: I guess that "N" probably has a misleading comment. I guess that I might have called these something else, but I don't think it is worth changing

**Review by brinxmat — APPROVED** [review]

File: cristin-import/src/main/java/no/unit/nva/cristin/MappingConstants.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package no.unit.nva.cristin;
+
+public final class MappingConstants {
+
+    public static String MAIN_CATEGORY_BOOK = "BOK";
```
Comment (janbj) [inline cristin-import/src/main/java/no/unit/nva/cristin/MappingConstants.java:5]: Do you have a source for this? Maybe in documentation for Cristin?

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java [hunk 3]
```diff
@@ -52,9 +62,44 @@ private EntityDescription generateEntityDescription() {
...
+        return new Reference.Builder()
+                   .withPublicationInstance(publicationInstance)
+                   .withPublishingContext(publicationContext)
+                   .build();
+    }
+
+    private PublicationContext buildPublicationContext() throws InvalidIsbnException {
+        if (isBook()) {
+            return new Book.Builder()
+                       .withIsbnList(Collections.emptyList())
```
Comment (janbj) [inline cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java:81]: We don't map ISBN?

**Review by janbj — APPROVED** [review]

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java (hunk 3 above)
Comment (axthosarouris) [reply cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java:81]: Fields are not yet available

File: cristin-import/src/main/java/no/unit/nva/cristin/MappingConstants.java (hunk 2 above)
Comment (axthosarouris) [reply cristin-import/src/main/java/no/unit/nva/cristin/MappingConstants.java:5]: I am in contact with Daniel to find the source that contains this information

**Pull request merged**